Description: Make bindgen an empty feature as zstd-sys patch mandates it
Last-Update: 2022-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -58,7 +58,7 @@
 
 [features]
 arrays = ["zstd-safe/arrays"]
-bindgen = ["zstd-safe/bindgen"]
+bindgen = []
 debug = ["zstd-safe/debug"]
 default = [
     "legacy",
